src/order/well_founded_set.lean

```diff
@@ -28,6 +31,9 @@ A well-founded subset of an ordered type is one on which the relation `<` is wel
   `set.add_antidiagonal` and `set.mul_antidiagonal` defined when `s` and `t` are well-founded.
 
 ## Main Results
+ * Higman's Lemma, `set.partially_well_ordered_on.partially_well_ordered_on_sublist_forall₂`,
```

Comment:
Would you mind adding a reference to the bib file? I guess to Higman's original article?